lib/bigqueryid/base.rb

```diff
@@ -13,21 +13,26 @@ module Base
     included do
       class_attribute :dataset_name
       class_attribute :table_name
+      class_attribute :table_partitioned
 
       def self.dataset(name)
         self.dataset_name = name
       end
 
-      def self.table(name)
+      def self.table(name, partitioned: false)
         self.table_name = name
+        self.table_partitioned = partitioned
       end
 
       def self.bigquery
         @bigquery_connection ||= Google::Cloud::Bigquery.new
       end
 
       def self.gcloud_table
-        bigquery.dataset(dataset_name).table(table_name)
+        table = table_name
+        table += "$#{Time.now.strftime("%Y%m%d")}" if table_partitioned
```

Comment:
De qualquer forma, não dá pra puxar essa informação do BigQuery de acordo com o contexto? Afinal, se for feito um insert a tabela de interesse é a da data atual... Mas se for pra pegar os dados de outro dia estaríamos pegando uma tabela da data correspondente, não?